### PR TITLE
Implementing environment variables

### PR DESCRIPTION
**Pull request by PacoDw**
The environment variables:
- MONGODB_ATLAS_PROJECT_ID 
- MONGODB_ATLAS_ORG_ID

### CODE REVIEW

File: mongodbatlas/data_source_mongodbatlas_clusters_test.go [hunk 1]
```diff
@@ -14,11 +15,11 @@ func TestAccDataSourceMongoDBAtlasClusters_basic(t *testing.T) {
 
 	resourceName := "mongodbatlas_cluster.test"
 	dataSourceName := "data.mongodbatlas_clusters.test"
-	projectID := "5cf5a45a9ccf6400e60981b6" // Modify until project data source is created.
+	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID") // Modify until project data source is created.
```
Comment (marinsalinas) [inline mongodbatlas/data_source_mongodbatlas_clusters_test.go]: Could you please remove this comment from all the test files?

File: mongodbatlas/data_source_mongodbatlas_clusters_test.go [hunk 2]
```diff
@@ -14,11 +15,11 @@ func TestAccDataSourceMongoDBAtlasClusters_basic(t *testing.T) {
...
 	name := fmt.Sprintf("test-acc-%s", acctest.RandString(10))
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
+		PreCheck:     func() { testAccPreCheck(t); checkProjectKeyEnv(t) },
```
Comment (marinsalinas) [inline mongodbatlas/data_source_mongodbatlas_clusters_test.go]: Could you please remove the` checkProjectKeyEnv` from here and put inside the `testAccPreCheck` function? In all test cases.

File: mongodbatlas/data_source_mongodbatlas_clusters_test.go (hunk 1 above)
Comment (PacoDw) [reply mongodbatlas/data_source_mongodbatlas_clusters_test.go]: yes, thank you 👍

File: mongodbatlas/data_source_mongodbatlas_clusters_test.go (hunk 2 above)
Comment (PacoDw) [reply mongodbatlas/data_source_mongodbatlas_clusters_test.go]: ok, thank you! 👍

File: mongodbatlas/provider_test.go [hunk 3]
```diff
@@ -26,7 +26,8 @@ func TestProvider(t *testing.T) {
 
 func testAccPreCheck(t *testing.T) {
 	if os.Getenv("MONGODB_ATLAS_PUBLIC_KEY") == "" ||
-		os.Getenv("MONGODB_ATLAS_PRIVATE_KEY") == "" {
-		t.Fatal("`MONGODB_ATLAS_PUBLIC_KEY` and `MONGODB_ATLAS_PRIVATE_KEY` mus be set for acceptance testing")
+		os.Getenv("MONGODB_ATLAS_PRIVATE_KEY") == "" ||
+		os.Getenv("MONGODB_ATLAS_PROJECT_ID") == "" {
```
Comment (marinsalinas) [inline mongodbatlas/provider_test.go]: Please add the validation for Organization ID

File: mongodbatlas/provider_test.go [hunk 4]
```diff
@@ -26,7 +26,8 @@ func TestProvider(t *testing.T) {
...
+		t.Fatal("`MONGODB_ATLAS_PUBLIC_KEY`, `MONGODB_ATLAS_PRIVATE_KEY`, `MONGODB_ATLAS_PROJECT_ID` and `MONGODB_ATLAS_ORG_ID` mus be set for acceptance testing")
```
Comment (marinsalinas) [inline mongodbatlas/provider_test.go]: change mus to must please :)

File: mongodbatlas/provider_test.go (hunk 3 above)
Comment (PacoDw) [reply mongodbatlas/provider_test.go]: thank you

**Pull request merged**